config/app.php

```diff
@@ -58,15 +58,11 @@
     */
 
     'debug_blacklist' => [
-        '_ENV' => [
-            'APP_KEY',
-            'DB_PASSWORD',
-        ],
+        '_COOKIE' => array_keys($_COOKIE),
```

Comment:
didnt understand logic?

Reply:
When you add an element to this array it replaces it's value with ************.

![screenshot-alibaba loc-2021 12 06-10_36_58](https://user-images.githubusercontent.com/45572261/144806126-bf111ad2-4b2e-46a0-9d23-e18c9c1e1955.png)

Reply:
i see but why array_keys($_COOKIE)

Reply:
it's my cookies then no problem acutally :)